src/components/SortControl/SortControl.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import './SortControl.css'
+
+interface SortControlProps {
+    currentSelection: string
+    onSortChange: (newSelection: string) => void
+}
+
+const SortControl: React.FC<SortControlProps> = ({
+    currentSelection,
+    onSortChange,
+}) => {
+    const options = ['Release Date', 'Title']
```

Comment:
If this value is constant, then it would be better to declare it outside of a React component to avoid recreating of an object with each re-render. Common practice is to move it to a separate file and import it from there.